libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -295,6 +296,16 @@ export function closeAllDropdowns({ type } = {}) {
   if (isDesktop.matches) setCurtainState(false);
 }
 
+export const toggleMenuMobile = (toggle) => {
+  if (!toggle) return;
+  const isExpanded = toggle.getAttribute('aria-expanded') === 'true';
+  toggle.setAttribute('aria-expanded', !isExpanded);
+  document.querySelector(selectors.navWrapper)?.classList?.toggle('feds-nav-wrapper--expanded', !isExpanded);
+  closeAllDropdowns();
+  setCurtainState(!isExpanded);
+  toggle.setAttribute('daa-ll', `hamburgermenu|${isExpanded ? 'open' : 'close'}`);
+};
+
```

Comment:
I'd argue that since this isn't used in multiple files, it can be left in the `global-navigation.js` one. You would then have direct access to `this.elements.navWrapper` and `toggle` and you'd keep the logic close to where it's being used.